components/automate-cli/pkg/reportgenerator/usagegenerator.go

```diff
@@ -175,9 +191,8 @@ func get(key string, s interface{}) interface{} {
 	return s.(map[string]interface{})[key]
 }
 
-func queryElasticSearchNodeReport(client *elastic.Client, startTime time.Time, endTime time.Time) {
-	filename := fmt.Sprintf("nodeinfo_%s_%s.csv", startTime.Format(timeFormat), endTime.Format(timeFormat))
-	f, err := os.OpenFile(filename, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0600)
+func queryElasticSearchNodeReport(client *elastic.Client, startTime time.Time, endTime time.Time, fileName string) {
+	f, err := os.OpenFile(fileName, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0666)
 	if err != nil {
 		fmt.Println("Failed to open Node Detail report")
```

Comment:
```suggestion
		logrus.Info("Failed to open Node Detail report")
```

Reply:
SInce this is CLI we cannot log to a log file @daveaugustus